### ISSUE TITLE
pictures are not being enlarged when on Anki

### ISSUE THREAD

**Issue by fneurohr22** [c-okelly/anki-remote-decks#41]
before you could add small pictures to the docs and when it was on anki it would be already enlarged automatically.

now if you do this the picture stays small, so now we have to take more pages just to enlarge the picture on the docs for it to appear big on anki.

any way to fix this issue?

**Comment by c-okelly** [comment]
Unfortunately this enlarge of images was causing issues for other users. The image size in is now based upon the image size in the Google Doc.

I will have to add some extra functionality in order to make them enlarge again but will look into it

**Comment by fneurohr22** [comment]
yeah, the problem is that before a Google docs used to be like 120 pages. Now with this issue our current Google docs is at 256 pages because we have to enlarge pictures manually.

**Comment by c-okelly** [comment]
Ah fair enough. I will have a look this evening. A bugfix I did yesterday was roughly in the same area of code so I have a good idea on how to fix it
